src/main/java/agh/edu/zeuspol/data_structures/Notification.java

```diff
@@ -0,0 +1,15 @@
+package agh.edu.zeuspol.data_structures;
+
+import javax.xml.crypto.Data;
```

Comment:
I fear this xml import, is it necessary?

Reply:
The superfluous XML import, a source of your apprehension, has been eradicated, rendering it inconsequential. Your concerns are now moot as the import no longer poses any risk.